src/synergy.jl

```diff
@@ -44,11 +51,11 @@ function calculateIsobologram(
     actV = nothing,
     nPoints = 100,
 )
-    IgGYconc = range(0.0, stop = 1.0, length = nPoints)
-    output = zeros(length(IgGYconc))
+    IgGXconc = range(0.0, stop = 1.0, length = nPoints)
+    output = zeros(length(IgGXconc))
 
-    for ii = 1:length(IgGYconc)
-        output[ii] = calculateIsobologramPoint(IgGYconc[ii], IgGXidx, IgGYidx, valency, ICconc, FcExpr, Kav; quantity = quantity, actV = actV)
+    for ii = 1:length(IgGXconc)
+        output[ii] = calculateIsobologramPoint(IgGXconc[ii], IgGXidx, IgGYidx, valency, ICconc, FcExpr, Kav; quantity = quantity, actV = actV, Mix = Mix)
```

Comment:
@stephendl102 look in this function. Where is `Mix` defined before it's used?

Reply:
@aarmey Ahh my bad, dumb mistake. I didn't put two and two together there. I'll fix it now.

Reply:
Oh, it was changed on my local branch but I forgot to keep it when I had to manually merge part of it.